### PR TITLE
DDF-5900 moved SubjectUtils, Permissions, and SecurityLogger to services

### PR DESCRIPTION
**Pull request by stustison**
@SmithJosh @garrettfreibott @blen-desta @bakejeyner @pklinef @rzwiefel

### CODE REVIEW

File: ui-backend/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/SearchFormsSymbolsIT.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public static void tearDownClass() {
 
   @Before
   public void setUp() {
-    when(MOCK_SUBJECT.isGuest()).thenReturn(false);
+    //    when(MOCK_SUBJECT.isGuest()).thenReturn(false);
```
Comment (blen-desta) [inline ui-backend/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/SearchFormsSymbolsIT.java]: ❓ Should this be deleted?

**Review by blen-desta — APPROVED** [review]
Hero'd with DDF PR. See https://github.com/codice/ddf/pull/6051#pullrequestreview-412220671

**Review by SmithJosh — APPROVED** [review]

File: ui-backend/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/SearchFormsSymbolsIT.java (hunk 1 above)
Comment (stustison) [reply ui-backend/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/SearchFormsSymbolsIT.java]: ah yeah probably

**Review by bakejeyner — APPROVED** [review]

**Comment by jlcsmith** [comment]
@stustison - DDF has been released so this should be unblocked.  Please update the dependency to DDF 2.25.1.

**Comment by vinamartin** [comment]
Merged in https://github.com/codice/ddf-ui/pull/282

**Pull request closed without merging**